src/main/scopes/js/get-access-path.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import * as ts from 'typescript'
+
+import type { Logger } from '../../core/log/logger.js'
+import { ComplexValue } from './complex-value.js'
+import { getNodeValueHandler } from './node-value-handler-map.js'
+
+/**
+ * Constructs a JsToken object from a given Identifier type AST node.
+ *
+ * @param node - TS node to retrieve access path for.
+ * @param sourceFile - Root AST node.
+ * @param logger - A logger instance.
+ * @param currAccessPath - For internal use only, tracks current constructed access path.
+ * @param topLevel - For internal use only, tracks top level function call.
```

Comment:
As a convention, since none of these params are optional, let's make `logger` be the last one in the function's params.